.github/workflows/codeql.yml

```
@@ -13,11 +13,6 @@ name: "CodeQL"
 
 on:
   push:
-  merge_group:
-    paths-ignore:
-    - "LICENSE"
-    - "CODEOWNERS"
-    - "**.md"
```

```suggestion
    paths-ignore:
    - "LICENSE"
    - "CODEOWNERS"
    - "**.md"
```

I think we still want to keep `paths-ignore` for [push events](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#onpushpull_requestpull_request_targetpathspaths-ignore). A change to these files does not require a re-run of codeql.

---

addresses in PR and agreed